server/profile.py

```diff
@@ -1,24 +1,37 @@
 import mysql.connector
 from mysql.connector import errorcode
 from connect import connectToDB
+from datetime import date
 
 
-def updateProfile(name, age, bio, gender, education, interests, genderPreference, id):
+def updateProfile(name, birthday_yr, birthday_mo, birthday_dt, bio, gender, education, interests, genderPreference, id):
     try:
         connection = connectToDB()
         if(connection != False):
             cursor = connection.cursor(buffered=True)
 
-            sql = '''INSERT INTO Profile (userId, firstname, age, description, gender, workplace, interests, genderPreference) 
+            sql = '''INSERT INTO Profile (userId, firstname,description, gender, workplace, interests, genderPreference, Birthday) 
                 VALUES (%s, %s, %s,%s,%s,%s,%s, %s);'''
-            values = (id, name, age, bio, gender, education,
-                      interests, genderPreference)
+            
+            birthday = date(int(birthday_yr), int(birthday_mo), int(birthday_dt))
+            values = (id, name, bio, gender, education,
+                      interests, genderPreference, birthday)
             cursor.execute(sql, values)
+
+            today = date.today()
+            age = today.year - birthday.year
+            if today.month < birthday.month or (today.month == birthday.month and today.day < birthday.day):
+                age -= 1
+
+            UpdateAgeQuery = "UPDATE Profile SET Age = %s WHERE userId = %s"
```

Comment:
Why you add this into your insert statement and set age along with your other properties?

Reply:
This was left-over cleaning up I need to do but I forgot. Originally I wrote another function under profile and later I combined them, this is the query pasted from that time, it will be cleaned up